jscomps/EmbedLiteConsoleListener.js

```diff
@@ -186,8 +166,8 @@ $EmbedLiteConsoleListener.prototype = {
         if (Logger.enabled) {
           this._listener = new SPConsoleListener();
           Services.console.registerListener(this._listener);
-          Services.obs.addObserver(this, "embedui:logger", true);
         }
+        Services.obs.addObserver(this, "embedui:logger", true);
```

Comment:
I assume this was intentional.  Can you describe briefly what effect this now has?  I.e. in the Logger.enabled=false case.

Reply:
Yes, intentional. We do not have front end for logging at the browser side (never have had such feature). That said, in order to have such a feature and being able to disable/enable logging from the front end, we need to be observing "embedui:logger" messages all the time regardless of the env variables.

Reply:
Would be good if such information could be added to the commit message at least

Reply:
Added:
```
After https://github.com/sailfishos/embedlite-components/pull/8 there is
no need to cache messages. This commit cleans up caches and related functions.

We do not have frontend for logging at the browser side (never have had
such feature). That said, in order to have such a feature and being able to
disable/enable logging from the front end, we need to be observing "embedui:logger"
messages all the time regardless of the EMBED_CONSOLE variable values.
```